proprietary/tokens/src/components/amsterdam/link.tokens.json

```diff
@@ -0,0 +1,95 @@
+{
+  "amsterdam": {
+    "link": {
+      "color": { "value": "{amsterdam.color.primary-blue}" },
+      "icon": {
+        "size": { "value": "1.2em" }
+      },
+      "text-decoration": { "value": "none" },
+      "text-underline-offset": { "value": "9px" },
+      "text-decoration-thickness": { "value": "2px" },
+      "text-decoration-skip-ink": { "value": "none" },
+      "font-family": {
+        "value": "{amsterdam.typography.font-family}"
+      },
+      "font-size": {
+        "narrow": { "value": "{amsterdam.typography.text-level.6.font-size.narrow}" },
+        "wide": { "value": "{amsterdam.typography.text-level.6.font-size.wide}" }
+      },
+      "font-weight": {
+        "value": "{amsterdam.typography.font-weight.normal}"
+      },
+      "focus": {
+        "color": { "value": "{amsterdam.color.dark-blue}" },
+        "text-decoration": { "value": "underline" }
+      },
+      "hover": {
+        "color": { "value": "{amsterdam.color.dark-blue}" },
+        "text-decoration": { "value": "underline" },
+        "text-decoration-thickness": { "value": "2px" }
+      },
+      "visited": {
+        "color": { "value": "{amsterdam.color.primary-blue}" }
+      },
+      "standalone-bold": {
+        "font-weight": {
+          "value": "{amsterdam.typography.font-weight.bold}"
+        }
+      },
+      "inline": {
+        "text-underline-offset": { "value": "3px" },
+        "font-family": {
+          "value": "inherit"
+        },
+        "font-size": {
+          "value": "inherit"
+        },
+        "visited": {
+          "color": { "value": "{amsterdam.color.purple}" }
+        }
+      },
+      "in-list": {
```

Comment:
Shouldn't this have text level tokens as well? The in list variant is smaller than the default link